src/status_im/data_store/realm/schemas/account/chat.cljs

```diff
@@ -251,3 +251,9 @@
   (update v13 :properties assoc
           :group-chat-local-version {:type :int
                                      :optional true}))
+(def v15
```

Comment:
could you elaborate on this?

Reply:
i don't see it used

Reply:
the only places these fields are mentioned anywhere in the source code is tests and realm schema. they are always `nil` as far as I'm concerned.

Reply:
I think what @flexsurfer means is that if you want these changes to take effect you also need to use the schema in the core namespace

Reply:
but  better remove these changes and make a seperate PR for it, because db changes are irreversible and this PR doesn't need them

Reply:
ah, yeah, forgot to attach them. sure, I will create a separate PR for them, makes sense.

Reply:
@yenda @flexsurfer https://github.com/status-im/status-react/pull/8315